.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
It's not that important for now, but in general those files should not necessarily be ignored, as we could also have non-generated `.pxi` files. Eg scipy includes the actual names of the generated file in the gitignore (because the generated files can also be eg `.pyx` files)